src/kern/npf_alg_pptp.c

```diff
@@ -135,8 +139,10 @@ typedef struct {
 
 #define	GRE_STATE_USED			0x1
 #define	GRE_STATE_ESTABLISHED		0x2
+#define	GRE_STATE_ORIGIN_CLIENT_CALL_ID		0x4
```

Comment:
Why is this flag needed? It seems that the original client ID is stored once the GRE state is created. In other words, `GRE_STATE_USED` implies `GRE_STATE_ORIGIN_CLIENT_CALL_ID`.